datadog/__init__.py

```diff
@@ -29,7 +29,7 @@
 logging.getLogger('datadog.threadstats').addHandler(NullHandler())
 
 
-def initialize(api_key=None, app_key=None, host_name=None, api_host=None,
+def initialize(api_key=None, app_key=None, host_name=None, hostname_from_config=True, api_host=None,
```

Comment:
I'd honestly put the new argument at the very end of the argument list, because it someone is doing right now `initialize("xxx", "yyy", None, "some_api_host")`, this would break their code. Optional arguments should always be added to the end.

Reply:
👍 Done in the last commit